### ISSUE TITLE
Trace model to onnx and inference using ONNXRuntime which can using TensorRT acceleration

### ISSUE THREAD

**Issue by jinfagang** [facebookresearch/maskrcnn-benchmark#998]
Does anyone have tried export model into onnx and inference using ONNXRuntime?

**Comment by lsy323** [comment]
Yes, I can export the model end-to-end to ONNX and inference using OnnxRuntime with this branch: https://github.com/t-vi/maskrcnn-benchmark/pull/3

**Comment by jinfagang** [comment]
@lsy323  That's awesome! I want to know how much improvements does onnx runtime got when inference?

**Comment by WonderAndMaps** [comment]
Are you able to run it with TensorRT? I can't run the onnx model I have or an official one in onnx model zoo.
